diff --git a/node_modules/@next-auth/prisma-adapter/dist/index.js b/node_modules/@next-auth/prisma-adapter/dist/index.js
index fc27cdd..d088c5c 100644
--- a/node_modules/@next-auth/prisma-adapter/dist/index.js
+++ b/node_modules/@next-auth/prisma-adapter/dist/index.js
@@ -35,7 +35,7 @@ const PrismaAdapter = (prisma) => {
                         return Promise.resolve(null);
                     return prisma.user.findUnique({ where: { email } });
                 },
-                async getUserByProviderAccountId(providerId, providerAccountId) {
+                async getUserByProviderAccountId({providerId, providerAccountId}) {
                     var _a;
                     const account = await prisma.account.findUnique({
                         where: {
@@ -62,7 +62,7 @@ const PrismaAdapter = (prisma) => {
                         where: { id: userId },
                     });
                 },
-                async linkAccount(userId, providerId, providerType, providerAccountId, refreshToken, accessToken, accessTokenExpires) {
+                async linkAccount({userId, providerId, providerType, providerAccountId, refreshToken, accessToken, accessTokenExpires}) {
                     await prisma.account.create({
                         data: {
                             userId,
